cypress/integration/basic-auth.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { datasetPage } from "../page-objects/app/datasets-page";
+
+describe("Basic browser auth", () => {
+
+  it("should login into webApp", () => {
+    datasetPage.visit();
+    datasetPage.leftPanel.leftPanelHeader.haveText('Datasets');
+  });
+
+  it("type in the input the name of a dataset", () => {
+    datasetPage.visit();
+    datasetPage.leftPanel.datasetsListElem.checkExists('xl_PGP3140_wgs_NIST-4_2');
```

Comment:
Your string parameter doing nothing. Please check the implementation of checkExists.

Reply:
Is it actual?

Reply:
fixed